enterprise/server/remote_execution/runner/runner_test.go

```diff
@@ -46,28 +47,42 @@ const (
 var (
 	// set by x_defs in BUILD file
 	testworkerRunfilePath string
+)
 
-	defaultCfg = &RunnerPoolOptions{
+func defaultCfg() *RunnerPoolOptions {
```

Comment:
was seeing some weird test errors that were caused by the `PoolOptions` subfields persisting across test runs. Fixed by making this a func so that each test generates a clean struct each time.